src/fiatconnect/ReviewScreenWrapper.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react'
+import { Screens } from 'src/navigator/Screens'
+import { StackScreenProps } from '@react-navigation/stack'
+import { StackParamList } from 'src/navigator/types'
+import { useSelector, useDispatch } from 'react-redux'
+import { StyleSheet, ActivityIndicator, View } from 'react-native'
+import colors from 'src/styles/colors'
+import {
+  fiatConnectQuotesErrorSelector,
+  cachedQuoteParamsSelector,
+} from 'src/fiatconnect/selectors'
+import variables from 'src/styles/variables'
+import { refetchQuote } from 'src/fiatconnect/slice'
+import { navigate } from 'src/navigator/NavigationService'
+
+type Props = StackScreenProps<StackParamList, Screens.FiatConnectReviewWrapper>
+
+export default function FiatConnectReviewScreenWrapper({ route }: Props) {
```

Comment:
Can this file / component be renamed too `RefetchQuoteScreen` or something like that? The page itself has nothing tying it to the ReviewScreen, its the saga that eventually navigates there. Neither is the screen a wrapper.

Reply:
Sure, definitely makes more sense.